preprocessors/openstreetmap/osm_service.py

```diff
@@ -154,6 +174,346 @@ def process_streets_data(OSM_data, bbox_coordinates):
         return processed_OSM_data
 
 
+def get_new_nodes(unbounded_node_list, node_list, bbox_coordinates):
+    # node_list1 contains the unbounded nodes
+    # node_list  contains the bounded nodes
+    node_list1 = unbounded_node_list
+    i = len(node_list)
```

Comment:
Is there a reason these need to be set to variables? If so, could you replace with better named variables?